### ISSUE TITLE
ERROR: Unable to locate Repeater in newest version of Burp Suite (v2022.1.1)

### ISSUE THREAD

**Issue by franjkovic** [b4dpxl/Burp-Repeater-Tab-Highlighter-Java#1]
Hi,

First, thanks for the awesome extension!

I am getting the following error in the Errors tab of the extension:
"ERROR: Unable to locate Repeater"

The following Exception is thrown: 
"java.lang.ClassCastException: class burp.cue cannot be cast to class javax.swing.JTabbedPane (burp.cue is in unnamed module of loader 'app'; javax.swing.JTabbedPane is in module java.desktop of loader 'bootstrap')"

The extension works properly in v2021.12.1.

**Comment by franjkovic** [comment]
Okay, so it seems the Repeater tab is now a JPanel that has a single JTabbedPane component.

To get the extension working again, I did the following hotfix: 

In [Highlighter.java](https://github.com/b4dpxl/Burp-Repeater-Tab-Highlighter-Java/blob/master/src/b4dpxl/tabhighlighter/Highlighter.java) I've added changed the following from:
```
                            if (t.getTitleAt(x).equalsIgnoreCase("Repeater")) {
                                this.repeater = (JTabbedPane) t.getComponentAt(x);
                                Utilities.debug("Found repeater :)");
                                return;
                            }
```

to
```
                            if (t.getTitleAt(x).equalsIgnoreCase("Repeater")) {
                            	JPanel temp = (JPanel) t.getComponentAt(x);
                            	this.repeater = (JTabbedPane) temp.getComponent(0);
                                Utilities.debug("Found repeater :)");
                                return;
                            }
```

on lines 91-95. I think there should probably be an if check there for instanceof JPanel vs instanceof JTabbedPane to cover older and newer versions of Burp Suite.

**Comment by b4dpxl** [comment]
Thanks for this, I hadn't actually updated yet. I'll implement a backwards-compatible fix and push a new release shortly.

**Comment by b4dpxl** [comment]
This should be fixed in the 1.3.0 release.
